lib/common/util.js

```diff
@@ -360,6 +362,14 @@ function makeAuthenticatedRequestFactory(config) {
     if (stream) {
       return stream;
     }
+
+    return {
+      abort: function() {
+        if (activeRequest_) {
+          activeRequest_.abort();
```

Comment:
hmm, would it actually make more sense to just return the `activeRequest_` instead of just wrapping `abort`? Or are there things we'd prefer not to expose?